internal/compiler/command/command.go

```diff
@@ -148,13 +166,30 @@ type (
 		// indicate a completely empty list.
 		Cols []Column
 	}
+
+	// Distinct skips datasets from the list that already have been encountered
+	// and returns a list with only distinct entries.
+	Distinct struct {
+		// Input is the input list that is filtered.
+		Input List
+	}
+
+	// Values returns a list of datasets from the evaluated expressions.
+	Values struct {
+		// Values are the values that represent the datasets in this list. Each
+		// dataset consists of all expressions that are in the dataset.
+		Values [][]Expr
```

Comment:
So this is a set of `parenthesizedExpr` in parser. But Im guessing that much context isn't needed here.

Reply:
What context? We don't need the information, that it was parenthesized expressions in the parser, no